src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -54,13 +65,16 @@ export interface SavedObjectsStartDeps {}
 export class SavedObjectsService
   implements CoreService<SavedObjectsServiceSetup, SavedObjectsServiceStart> {
   private migrator: KibanaMigrator | undefined;
-  logger: Logger;
+  private logger: Logger;
+  private clientProvider:
+    | ScopedSavedObjectsClientProvider<KibanaRequest<unknown, unknown, unknown>>
```

Comment:
super nit: `KibanaRequest<unknown, unknown, unknown>` --> KibanaRequest`, `unknown` is the default type.